Model/Config/ThirdPartyPayment.php

```diff
@@ -37,17 +37,18 @@ public function isAllowed(int $storeId = null): bool
 
     /**
      * @param int|null $storeId
-     * @return string|null
+     * @return array
      */
-    public function getAllowedMethod(int $storeId = null): ?string
+    public function getAllowedMethods(int $storeId = null): array
     {
         $allowedThirdPartyValue = $this->scopeConfig->getValue(
             'swarming_subscribepro/third_party_payment/allowed_method',
             ScopeInterface::SCOPE_STORE,
             $storeId
         );
+        $allowedThirdPartyArray = explode(',', $allowedThirdPartyValue);
 
-        return $this->isAllowed($storeId) ? $allowedThirdPartyValue : null;
+        return $this->isAllowed($storeId) ? $allowedThirdPartyArray : [];
```

Comment:
We should take into account the case when `$allowedThirdPartyValue` is a null or empty string.
If it's null then an error will occur, since the `explode` method expects a string.
If it's an empty string, then `$allowedThirdPartyArray` will contain an array with one element - an empty string.

Reply:
This satisfies all the possbile types: empty string, integer, null